{{ cookiecutter.project_slug }}/docker/Dockerfile

```diff
@@ -17,4 +17,6 @@ RUN pip install -r /requirements.txt
 
 ARG UID
 RUN useradd docker -l -u $UID -s /bin/bash -m
-USER docker
\ No newline at end of file
+USER docker
+
+ENV PATH="/home/docker/.local/bin:${PATH}"
```

Comment:
I think this line is not needed for this pull request (Makes data_source variable optional ).